### PR TITLE
[T4A2][W13-B1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**
Use class level members

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Ready to review

**Comment by okkhoy** [comment]
@LuMiN0uSaRc Shouldn't the team ID be B1? Please correct it!

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

Please note, going forward, we will not review PRs that are not clean!

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -16,6 +16,8 @@
     private Address address;
 
     private final UniqueTagList tags;
+    private int sequenceNumber;
+    public static int nextSequenceNumber = 0;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Person.java]: think why it may be appropriate to initialize it to 1 rather than 0.

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Some comments added. Please ack & close the PR after reading comments.

**Comment by LuMiN0uSaRc** [comment]
Noted with thanks!

**Pull request closed without merging**